src/main/java/frc/robot/turret/simulation/SimulationConstants.java

```diff
@@ -1,9 +1,14 @@
 package frc.robot.turret.simulation;
 
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
 import edu.wpi.first.math.controller.PIDController;
 
 public class SimulationConstants {
     public static double GEARING = 50.0;
     public static int AMOUNT_OF_MOTORS = 1;
     public static double JKG_METER_SQ = 0.00048;
+    public static TalonFXConfiguration MOTOR_CONFIGURATION = new TalonFXConfiguration();
+    static {
+        MOTOR_CONFIGURATION.Slot0.kP = 1;
+    }
```

Comment:
I'm guessing you don't have any other option but I just wanted to say I'm not a fan of this

Reply:
im also not a fan of this but its the only way so ill keep it like that

Reply:
this is not the only way.
you can do `new TalonFXConfiguration().withSlot0(new Slot0Configs().withKP(1));`